pkgs/applications/science/physics/freefem/default.nix

```diff
@@ -0,0 +1,82 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, autoreconfHook
+, bison
+, flex
+, unzip
+, gfortran
+, lapack
+, hdf5-cpp
+, scotch
+, fftw
+, arpack
+, suitesparse
+, nlopt
+, ipopt
+, gmm
+, tetgen
+, gnuplot_qt
+}:
+stdenv.mkDerivation rec {
+  pname = "freefem";
+  version = "4.12";
+
+  src = fetchFromGitHub {
+    owner = "FreeFem";
+    repo = "FreeFem-sources";
+    rev = "v${version}";
+    sha256 = "sha256-6EdkXaWf76wODfKwKw2gKRLY9hpcHUzAgtTSdQeHSoU=";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+    bison
+    flex
+    unzip
+    gfortran
+  ];
+
+  buildInputs = [
+    lapack
+    hdf5-cpp
+    scotch
+    fftw
+    arpack
+    suitesparse
+    nlopt
+    ipopt
+    gmm
+    tetgen
+    gnuplot_qt
+  ];
+
+  outputs = [ "out" ];
+  enableParallelBuilding = true;
+
+  configureFlags = [ "--enable-optim" ];
+
+  # Currently this fails with 1 test
+  # doCheck = true;
```

Comment:
Can we disable just that one failing test.

Reply:
Fixed in current version